source/Ship.cpp

```diff
@@ -1088,7 +1112,9 @@ void Ship::Save(DataWriter &out) const
 		}
 		for(const Hardpoint &hardpoint : armament.Get())
 		{
-			const char *type = (hardpoint.IsTurret() ? "turret" : "gun");
+			// This sets *type to one of turret, pylon, or gun based on which hardpoint.Is_____ it matches.
```

Comment:
```suggestion
			// This sets *type to one of turret, pylon, or gun, based on which hardpoint it matches.
```

Reply:
This one is more of a coding note referring to `IsGun`, `IsTurret`, `IsPylon`. I'm going to leave it in for now until I'm confident I have all the changes I'm going to be making.

One additional item planned to go alongside this is changing how guns are handled. Currently there's a moderate number of "Do X if (not a turret and not a pylon), since if it's not a turret or pylon it must be a gun" logic. This is part of changing that to "Do X if this is a gun" process. A reminder of sorts.

Reply:
Should there at least be a space?
```suggestion
			// This sets *type to one of turret, pylon, or gun based on which hardpoint Is_____ it matches.
```